app/helpers/transam_format_helper.rb

```diff
@@ -215,9 +215,9 @@ def format_as_date_time(datetime, compact=true)
 
   def format_as_date(date, compact=true)
     if compact
-      date.strftime("%m/%d/%Y") unless date.nil?
+      date.strftime("%m/%d/%Y") unless (date.nil? || date == Date.new(1))
```

Comment:
I really dislike creating garbage just for a simple comparison like this. I would prefer either creating a constant to be compared against, or ideally, checking some standard attribute. In this case you could do `date.year == 1`.